lib/file-source/src/lib.rs

```diff
@@ -235,13 +245,9 @@ mod test {
                 }
                 FWAction::RotateFile => {
                     let mut new_path = path.clone();
-                    new_path.set_extension("log.1");
-                    match fs::rename(&path, &new_path) {
-                        Ok(_) => {}
-                        Err(_) => {
-                            assert!(false);
-                        }
-                    }
+                    new_path.set_extension(format!("log.{}", rotation_count));
```

Comment:
I'm sure it doesn't matter, just noting that this is a bit different from how `logrotate` works now. Bummer that windows doesn't work that way.

Reply:
Yeah, there were some other options but this seemed the safest.